languages/en.ini

```diff
@@ -172,6 +172,8 @@ Catalog Results = "Catalog Results"
 catalog_login_desc = "Enter your library catalog credentials."
 CD = "CD"
 Change Password = "Change Password"
+channel_expand_icon = "Show more channels like this one"
+channel_search_icon = "Show channel as search results"
```

Comment:
We still need to think about this language. It is particularly problematic for the search_icon link, which does not go to search results but rather loads the record that inspired the channel. I wonder if expand_icon would be more accureate if it said "related to this one" rather than "like this one."